pages/agent/cli_pipeline.md.erb

```diff
@@ -115,49 +115,103 @@ available for `master` builds:
   branches: "master"
 ```
 
-Add a prompt to the `block` step by adding the `prompt` and `fields` configuration keys:
+Adding the `fields` attribute to a *block* step allows you to add custom fields to the prompt when you click on a *block* step:
 
 ```yml
-- block: "Continue after input"
-  prompt: "Fill out this form!"
-  fields: 
-    - text: "What version is this?"
-      key: "version"
-      default: "v0.1"
+- block: Release
+    prompt: Fill out the details for release
+    fields: 
+      - text: Release notes
+        key: release-notes
+        hint: Enter your release notes here \:sparkles\:
+      - text: Release name
+        key: release-name
+        optional: true
+      - select: Release type
+        key: release-type
+        default: Major
+        options:
+          - Major
+          - Minor
+          - Patch
 ```
 
-There are two types of fields available: `text`, and `select`. In a `select` field, the number of `options` specified will alter its display in the prompt. Providing 6 options or less will show as a set of radio buttons, and more than 6 options will show a drop down list.  
-
-```yml
-- block: "Continue after input"
-  prompt: "Fill out this form!"
-  fields: 
-    - select: "What team should be notified after this deployment?"
-      key: "notify-team"
-      options:
-        - All
-        - None
-        - Billing
-        - Ops
-        - Sales
-        - Directors
-        - Engineering
-        - Support
-        - QA
-```
+You can test out a live example by adding our sample pipeline with an unblock prompt step to your existing Buildkite organisation: 
+
+<a class="inline-block" href="/new?template=https://github.com/buildkite/unblock-prompt-example" target="__blank"><img src="https://buildkite.com/button.svg" alt="Add Unblock Prompt Example to Buildkite" class="no-decoration" width="160" height="30"></a>
+
+## block attributes
+
+Attributes that are available to describe block steps in your `pipeline.yml`:
+
+<table>
+  <tr>
+    <th>Attribute</th>
+    <th>Example</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>name</td>
+    <td>Release \:rocket\:</td>
+    <td>the text that is displayed for the step in the pipeline</td>
+  </tr>
+  <tr>
+    <td>branches</td>
+    <td>master</td>
+    <td>
+      scope the block step to a set of branches <br>
```

Comment:
Spaces - we've got a doc you can link to: https://buildkite.com/docs/pipelines/branch-configuration